Fix MyPy linting

Currently failing because [mypy 0.9.0 unbundled third-party type stubs](https://mypy-lang.blogspot.com/2021/06/mypy-0900-released.html), which now need to be installed separately.

```
mypy .
pylsp_black/plugin.py:4: error: Library stubs not installed for "toml" (or incompatible with Python 3.8)
pylsp_black/plugin.py:4: note: Hint: "python3 -m pip install types-toml"
pylsp_black/plugin.py:4: note: (or run "mypy --install-types" to install all missing stub packages)
pylsp_black/plugin.py:4: note: See https://mypy.readthedocs.io/en/stable/running_mypy.html#missing-imports
tests/test_plugin.py:8: error: Library stubs not installed for "pkg_resources" (or incompatible with Python 3.8)
tests/test_plugin.py:8: note: Hint: "python3 -m pip install types-pkg_resources"
Found 2 errors in 2 files (checked 6 source files)
```

--- a/setup.cfg
+++ b/setup.cfg
@@ -21,7 +21,7 @@ python_requires = >= 3.6
 pylsp = pylsp_black = pylsp_black.plugin
 
 [options.extras_require]
-dev = isort>=5.0; flake8; pytest; mypy; pytest
+dev = isort>=5.0; flake8; pytest; mypy; pytest; types-pkg_resources; types-toml
 
 [flake8]
 max-line-length = 88

